README.markdown

```diff
@@ -168,6 +168,7 @@ API Implemented
 * [ngx.req.set_method](https://github.com/openresty/lua-nginx-module#ngxreqset_method)
 * [ngx.req.set_header](https://github.com/openresty/lua-nginx-module#ngxreqset_header)  (partial: table-typed header values not supported yet)
 * [ngx.req.clear_header](https://github.com/openresty/lua-nginx-module#ngxreqclear_header)
+* [ngx.req.get_uri_exten](https://github.com/openresty/lua-nginx-module#ngxreqget_uri_exten)
```

Comment:
This is wrong since it should be a brand new Lua module. This document section documents *existing* ngx_lua CFunction-based Lua APIs re-implemented via FFI in this library.